src/MICore/MICommandFactory.cs

```diff
@@ -617,16 +617,6 @@ public override bool SupportsStopOnDynamicLibLoad()
             return false;
         }
 
-        public override Results IsModuleLoad(string cmd)
```

Comment:
Should we make the base method non-virtual now that we don't have any overrides of it?